src/BlazorWebView/src/WindowsForms/BlazorWebView.cs

```diff
@@ -144,7 +134,13 @@ private void StartWebViewCoreIfPossible()
 		{
 			// We never start the Blazor code in design time because it doesn't make sense to run
 			// a Blazor component in the designer.
-			if (!IsAncestorSiteInDesignMode2 && (!RequiredStartupPropertiesSet || _webviewManager != null))
+			if (IsAncestorSiteInDesignMode)
```

Comment:
This is the crux of the change. Switch from `IsAncestorSiteInDesignMode2` (note the `2` suffix) to the built-in `IsAncestorSiteInDesignMode2`. And more importantly, flip the `!` logic so that the initialization is _not_ done when the ancestor is in design mode.